message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1556,5 +1556,33 @@
       <field name="AOA" type="float" units="deg">Angle of Attack (degrees)</field>
       <field name="SSA" type="float" units="deg">Side Slip Angle (degrees)</field>
     </message>
+    <message id="11030" name="ESC_TELEMETRY_1_TO_4">
+      <description>ESC Telemetry Data for ESCs 1 to 4, matching data sent by BLHeli ESCs</description>
+      <field type="uint8_t[4]" name="temperature">Temperature (C)</field>
+      <field type="uint16_t[4]" name="voltage">Voltage (Volts * 0.01)</field>
+      <field type="uint16_t[4]" name="current">Current (Amps * 0.01)</field>
```

Comment:
This seems to be tight for future usage, I could easily see ESC's being available that can exceed this field.

Reply:
650A per ESC? Really?

Reply:
Sorry I apparently can't reliably do math in my head before breakfast, 650 should be fine :)